TOOLS/lua/autocrop.lua

```diff
@@ -18,97 +18,225 @@
 -- is unambiguous (i.e., not a black frame, black background title
 -- card, or dark scene).
 --
--- The default delay between insertion of the cropdetect and
--- crop filters may be overridden by adding
---
--- --script-opts=autocrop.detect_seconds=<number of seconds>
---
--- to mpv's arguments. This may be desirable to allow cropdetect more
--- time to collect data.
+-- The default options can be overridden by adding
+-- --script-opts-append=autocrop-<parameter>=<value>
+-- into mpv.conf
```

Comment:
You should make the syntax consistent with `mpv.conf` if you want to change this. There shouldn't be the preceding "--" in that case. You should also move the description of the options here in the header because they might get missed if they are in the body of the script.